websocketpp/common/security_context.hpp

```diff
@@ -37,9 +37,14 @@
 
 #ifdef _WIN32
 #include <winapifamily.h>
-#endif
 
-#if defined(_WIN32) && WINAPI_FAMILY_PARTITION(WINAPI_FAMILY_DESKTOP)
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)
```

Comment:
I think this means Win32, not UWP 
https://git.corp.adobe.com/torq/torq-native/blob/develop/modules/api/public/torq/api/TorqDefines.h#L66

Reply:
Renamed to WEBSOCKETPP_WIN32